lib/auth0/api/v2/users.rb

```diff
@@ -30,39 +50,107 @@ def create_user(name, options = {})
-        def user(user_id, fields: nil)
+        # Retrieves a user given a user_id
+        # @see https://auth0.com/docs/api/v2#!/Users/get_users_by_id
+        # @param user_id [string] The user_id of the user to retrieve
+        # @param fields [string] A comma separated list of fields to include or exclude from the result.
+        # @param include_fields [boolean] if the fields specified are to be included in the result, false otherwise.
+        #
+        # @return [json] the user with the given user_id if exists
+        def user(user_id, fields: nil, include_fields: true)
+          fail Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
           path = "/api/v2/users/#{user_id}"
           request_params = {
-            fields:         fields
+            fields:         fields,
+            include_fields: include_fields
           }
           get(path, request_params)
         end
 
-        # https://auth0.com/docs/apiv2#!/users/delete_users_by_id
+        # Deletes a single user given its id
+        # @see https://auth0.com/docs/api/v2#!/Users/delete_users_by_id
+        # @param user_id [string] The user_id of the user to delete
         def delete_user(user_id)
-          fail Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.nil? || user_id.to_s.empty?
+          fail Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
           path = "/api/v2/users/#{user_id}"
           delete(path)
         end
 
-        # https://auth0.com/docs/apiv2#!/users/patch_users_by_id
+        # Updates a user with the object's properties received in the optional parameters.
+        # These are the attributes that can be updated at the root level:
+        # blocked, email_verified, email, verify_email, password, phone_number, phone_verified,
+        # verify_password, user_metadata, app_metadata, username
+        # Some considerations:
+        # The properties of the new object will replace the old ones.
+        # The metadata fields are an exception to this rule (user_metadata and app_metadata). These properties are
+        # merged instead of being replaced but be careful, the merge only occurs on the first level.
+        # If you are updating email_verified, phone_verified, username or password you need to specify the connection
+        # property too.
+        # If your are updating email or phone_number you need to specify the connection and the client_id properties.
+        # @see https://auth0.com/docs/api/v2#!/Users/patch_users_by_id
+        # @param user_id [string] The user_id of the user to update.
+        # @param options [hash] The optional parametes to update
+        #
+        # @return [json] the updated user
         def patch_user(user_id, options)
+          fail Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
+          fail Auth0::InvalidParameter, 'Must supply a valid body' if options.to_s.empty?
```

Comment:
Message says Must supply body and parameter is options. Is this message set by requirement?